src/utils/data.ts

```diff
@@ -33,8 +33,8 @@ export const getTasks = (): TaskDict => {
   return results;
 };
 
-export const getTeamMemberByGithubUsername = (github_username: string): TeamMember | undefined => {
-  let member: TeamMember | undefined;
+export const getTeamMemberByGithubUsername = (github_username: string): TeamMember | null => {
+  let member: TeamMember | null = null;
```

Comment:
hmmm what does `null = null` do here?

Reply:
its saying that `member`, which can be of type `TeamMember` or `null`, will be initialized as `null`

Reply:
i basically added this because when we `return` the `member` variable at the end of this function, my IDE was complaining that it might not be initialized